frontend/src/components/pages/gamecat.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useRef, useState } from 'react';
+import { useLocation, Link } from 'react-router-dom';
+import { BrowserRouter as Router, Route, Routes, useNavigate } from 'react-router-dom';
+
+export default function GameCategory(prop) {
+    const [iteration, setIteration] = React.useState(prop.iteration);
+    const [category, setCategory] = React.useState(prop.cat);
+    const [gameinfo, setGameinfo] = React.useState(prop.gameinfo);
+
+    let cats = [
```

Comment:
Can't you get the category ids from the API to set them dynamically?

Reply:
i tried, and no, not with the way the ids are set up. It's easier just to set it manually